src/PostgREST/Query/SqlFragment.hs

```diff
@@ -252,7 +252,7 @@ pgFmtCallUnary :: Text -> SQL.Snippet -> SQL.Snippet
 pgFmtCallUnary f x = SQL.sql (encodeUtf8 f) <> "(" <> x <> ")"
 
 pgFmtField :: QualifiedIdentifier -> CoercibleField -> SQL.Snippet
-pgFmtField table CoercibleField{cfFullRow=True}                                          = fromQi table
+pgFmtField table CoercibleField{cfFullRow=True}                                          = pgFmtIdent (qiName table)
```

Comment:
This was working on to-one relationships because it was always aliased. For many-to-many it gives an error since it's not aliased and it includes the schema, when it shouldn't:

```
-- SQL error
SELECT "test"."tableA" AS count
FROM "test"."tableA", "test"."tableB"
...
```

This fixes that, it only uses the table name or alias.